types/src/color.d.ts

```diff
@@ -28,13 +28,19 @@ export interface ColorObject {
 	alpha?: number | undefined;
 }
 
+export interface PlainColorObject {
+	space: ColorSpace;
+	coords: Coords;
+	alpha: number;
+}
+
 export interface ColorConstructor {
 	spaceId: string;
 	coords: Coords;
 	alpha: number | undefined;
 }
 
-export type ColorTypes = Color | ColorObject | ColorConstructor | string;
+export type ColorTypes = ColorObject | ColorConstructor | string | PlainColorObject;
```

Comment:
Was it intended to remove `Color` from `ColorTypes` here?

Reply:
Yes, ```Color``` implements the ```PlainColorObject``` interface so a ```Color``` is always a ```PlainColorObject```.